app/novelty/compute_novelty.py

```diff
@@ -3,10 +3,22 @@
 import httpx
 import numpy as np
 import traceback
-
-from .known import find_known_results
-from .extr_smile_molpro_by_id import mol_to_smile_molpro
-from .mol_similarity import find_nearest_neighbors
+import json
+from rdkit import Chem
+from rdkit import DataStructs
+from rdkit.Chem import AllChem
+import os, sys
+import time
+import asyncio
+import requests
+import redis
+
+startup_utils_path = os.path.abspath("app/novelty/gene_nmf_adapter.py")
```

Comment:
Again, adding files to the system path shouldn't be needed here. These can just be imported since they're within the Appraiser "package". Please remove all system path logic.